courses/free-courses-en.md

```diff
@@ -603,6 +603,7 @@
 
 * [Beginner's Series to: Node.js](https://youtube.com/playlist?list=PLlrxD0HtieHje-_287YJKhY8tDeSItwtg) - Microsoft Developer (YouTube)
 * [Building a RESTful API with Node.js](https://www.youtube.com/playlist?list=PL55RiY5tL51q4D-B63KBnygU6opNPFk_q) - Academind (YouTube)
+* [Data brokering with Node.js: Process data at the speed of technology](https://heynode.com) - Osio Labs Inc. *(signup required, not requested)*
```

Comment:
`signup required, not requested` ???

Reply:
reverting commit feb84cae5b8537eb3c7432a47556cd24e04d8300

Reply:
it's obligatory to register

Reply:
> it's obligatory to register

You are right, but is also a fact that each chapter of all tutorials has a intro text that is free and doesn't require register to read it. Definitely, if you are interested in contents, you need an user account in the website

Fitting options:
- signup required
- signup requested, not required

Luigi, the last suggestion not is a common resource note reflecting access notes guidelines